docs/developers-guide/substreams-sink-files.md

```diff
@@ -0,0 +1,220 @@
+2023-01-09T07:45:04.048-0800 INFO (substreams-sink-files) starting new file boundary {"boundary": "[0, 10000)"}
+2023-01-09T07:45:04.049-0800 INFO (substreams-sink-files) boundary started {"boundary": "[0, 10000)"}
+2023-01-09T07:45:04.049-0800 INFO (substreams-sink-files) starting stats service {"runs_each": "2s"}
+2023-01-09T07:45:06.052-0800 INFO (substreams-sink-files) substreams sink stats {"progress_msg_rate": "0.000 msg/s (0 total)", "block_rate": "650.000 blocks/s (1300 total)", "last_block": "#1299 (a0f0f283e0d297dd4bcf4bbff916b1df139d08336ad970e77f26b45f9a521802)"}
+```
+
+## Conclusion and review
+
+The ability to route data extracted from the blockchain by using Substreams is powerful and useful. Files aren't the only type of sink the data extracted by Substreams can be piped into. Review the core Substreams sinks documentation for [additional information on other types of sinks](https://substreams.streamingfast.io/developers-guide/substreams-sinks) and sinking strategies.
+
+### Recap
+
+To use `substreams-sink-files` you need to clone the official repository, install the tooling, generate the required files from the substreams CLI for the example Substreams module and run the sink tool.
+
+You have to ensure the sinking strategy has been defined, the appropriate file types have been targeted, and accounted for, and the module handler code in your Substreams module has been properly updated. You need to start the `substreams-sink-files` tool and use the `run` command being sure to provide all of the required values for the various flags.
+
+---
+
+**-- QUESTIONS --**
+
+**3. QUESTION**: 'There need to have some content about how the limitation of this sink which write bundles only when last block of a bundle is final.'
+
+<i>I need input on this or some type of reference or something to find information for what we want to tell the reader.</i>
+
+**4. QUESTION**: 'Add discussion about where Substreams cursor is saved and importance of persisting this state (save as a .yaml file).'
+
+<i>I need input on this or some type of reference or something to find information for what we want to tell the reader.</i>
+
+**5. QUESTION**:
+'Add discussion about s3, gcs, etc. vs. local files'
+
+<i>Where do we want to put this in the document, what section?</i>
+
+'A local folder
+A Google Cloud Storage Bucket (gs://<bucket>/<path>)
+An S3 compatible Bucket (s3://<bucket>/<path>)
+An Azure bucket (az://<bucket>/<path>)
+Configuration details for those could be seen at https://github.com/streamingfast/dstore#features'
+
+**6. QUESTION:**
+
+- 'output'
+  <i>how to show the directories and files that get produced, is the logging to the terminal sufficient? The Google style guide restricts using screenshots so that limits using a screen capture of the directories and files that get produced. Should we describe the output somehow? Please advise.</i>
+- 'inspect'
+  <i>I need input for this, not sure what it is.
+  </i>
```

Comment:
Describing the output is enough, no screenshot needed. You could should the output of `ls <storagePath>` and limited the displayed output to a few files. Then the a quick description of a bundle could be given, like the block range it spans and the actual content (all the lines received for this 10K blocks by default, controlled by `--file-block-count`).

Reply:
In progress.

Reply:
Content added.